### PR TITLE
feat: New tooltip preparation

### PR DESCRIPTION
**Pull request by przemvs**
https://wearezeta.atlassian.net/browse/WPB-6320

## Description

Created new tooltips which is visible all of the time whenever You hover element with tooltip. Now tooltips are not cutted.
Tooltips works dynamically, if on top will not place for tooltip, will be placed under hovered element.

Also version bump for @wireapp/react-ui-kit from 9.12.8 to 9.15.1 (New tooltips included).

## Screenshots/Screencast (for UI changes)

image

image

image

image

## Checklist

- [ ] PR has been self reviewed by the author;
- [ ] Hard-to-understand areas of the code have been commented;
- [ ] If it is a core feature, unit tests have been added;

### Important details for the reviewers

(Delete this section if unnecessary)

- use (x) data
- can be reviewed commit-by-commit
- be sure to look at ...

### CODE REVIEW

File: src/script/components/MessagesList/Message/ContentMessage/MessageHeader.tsx [hunk 1]
```diff
@@ -49,33 +51,25 @@ function BadgeSection({sender}: {sender: User}) {
...
-        >
+        <Tooltip className="message-header-icon-guest" body={sender.handle} data-uie-name="sender-federated">
           <Icon.Federation />
-        </span>
+        </Tooltip>
       )}
 
       {sender.isDirectGuest() && !sender.isFederated && (
-        <span
-          className="message-header-icon-guest with-tooltip with-tooltip--external"
```
Comment (atomrc) [inline src/script/components/MessagesList/Message/ContentMessage/MessageHeader.tsx:73]: could we also delete those css declarations? Are they still used?

**Review by atomrc — APPROVED** [review]
nice 😎

File: src/script/components/MessagesList/Message/ContentMessage/MessageHeader.tsx (hunk 1 above)
Comment (przemvs) [reply src/script/components/MessagesList/Message/ContentMessage/MessageHeader.tsx:73]: Yes, this class is used :) 
E.g. with class:
image

without class:
image

Comment (atomrc) [reply src/script/components/MessagesList/Message/ContentMessage/MessageHeader.tsx:73]: so they are still used in the code, right

**Pull request merged**